eng/pipelines/templates/steps/build-test.yml

```diff
@@ -55,17 +55,29 @@ steps:
 
   - ${{ parameters.AfterTestSteps }}
 
+  - pwsh: |
+      Get-ChildItem .\_coverage\ |
+      Foreach-Object {
+        Get-Content $_
+      }
+    displayName: 'Show .coverage files'
+    condition: and(succeededOrFailed(), ${{ parameters.RunCoverage }})
+
+  - task: PublishPipelineArtifact@1
+    displayName: 'Publish .coverage files'
+    continueOnError: true
+    condition: and(succeededOrFailed(), ${{ parameters.RunCoverage }})
+    inputs:
+      targetPath: '$(Build.SourcesDirectory)/_coverage'
```

Comment:
Does this work? DevOps generally doesn't like you uploading an artifact with the same name from multiple jobs/stages. Usually you have to have a unique name.

Reply:
Yes it uploads the artifacts in a folder named after the job so each job is unique.